src/omnicore/test/alert_tests.cpp

```diff
@@ -79,7 +79,7 @@ BOOST_AUTO_TEST_CASE(alert_authorize_any_source)
     mapMultiArgs = mapMultiArgsOriginal;
     mapArgs = mapArgsOriginal;
 }
-
+/*
```

Comment:
Unless this is only temporary, I think we may just remove commented code completely.